app/src/main/res/values/strings.xml

```diff
@@ -44,12 +44,19 @@
     <string name="label_user_deleted">Deleted</string>
     <string name="label_message_sent_failure">Message could not be sent due to connectivity issues.</string>
     <string name="label_message_edit_sent_failure">The edited message could not be sent due to connectivity issues.</string>
-    <string name="label_message_sent_remotely_failure">Message could not be sent, as the backend %s appears to be offline.</string>
-    <string name="label_message_edit_sent_remotely_failure">The edited message could not be sent, as the backend %s appears to be offline.</string>
+    <string name="label_message_sent_remotely_failure">Message could not be sent, as the backend of %s could not be reached.</string>
+    <string name="label_message_edit_sent_remotely_failure">The edited message could not be sent, as the backend of %s could not be reached.</string>
     <string name="label_message_receive_failure">Download Error</string>
     <string name="label_message_decryption_failure_message">Message could not be decrypted.</string>
     <string name="label_message_decryption_failure_informative_message">Try resetting the session to generate new encryption keys.</string>
     <string name="label_message_knock">%s pinged</string>
+    <string name="label_message_partial_delivery_participants_count">%1$d participants didn\'t get your message.</string>
```

Comment:
maybe this should also be a plural string? 
for 1 person it should be "1 participant didn't get your message" 😄

Reply:
Makes sense actually, the only thing, is that designs for this, we are not showing counter total if it's only one participant 😓